src/Data/Vec/Properties.agda

```diff
@@ -369,10 +369,15 @@ map-id : ∀ {n} → map {A = A} {n = n} id ≗ id
 map-id []       = refl
 map-id (x ∷ xs) = P.cong (x ∷_) (map-id xs)
 
-map-const : ∀ {n} (xs : Vec A n) (y : B) → map {n = n} (const y) xs ≡ replicate y
+map-const : ∀ {n} (xs : Vec A n) (y : B) → map (const y) xs ≡ replicate y
 map-const [] _ = refl
 map-const (_ ∷ xs) y = P.cong (y ∷_) (map-const xs y)
 
+map-++-commute : ∀ {m} {n} (f : A → B) (xs : Vec A m) (ys : Vec A n) →
```

Comment:
Fair enough. As with some of the other style comments above, I think I found myself attempting, not always faithfully, to emulate the naming conventions of `Data.List.Properties`... and I confess I'd prefer short names if possible, but your remark about `reverse-foldr` etc. had me wondering, in such a 'commute'/'complex=>simple' unfolding, which order even the commuting operations should be named. I had also wondered whether (see `Data.Vec.Base` for example) whether reorgansing this file so that everything was stated in terms of `foldr` and `foldl` might be a good idea, or a terrible one...

Reply:
Hmm. Originally, there was also a `sum-++-commute`... another deprecation opportunity?

Reply:
Yes, in a perfect world we would be consistent about the ordering as well. I think lets hold off any reorganisation in terms of `foldr` and `foldl` for at least in this PR!